website/src/components/assets/images/svg.js

```diff
@@ -10,6 +10,7 @@ export const CheckIcon = (props) => {
       fill={fill}
       xmlns="http://www.w3.org/2000/svg"
       viewBox="0 0 24 24"
+      focusable="false"
```

Comment:
Hva gjør `focusable="false"` her? Er det sånn at vi må ha det i `@navikt/ds-icons` også?

Reply:
Tenker det er opp til bruker å sette selv hvis den ønsker.
Dette har ingen effect for det meste, men kan føre til litt merkelige interaksjoner. I dette tilfellet er det bare for å fikse warnings fra `ARC toolkit` a11y extension.